### PR TITLE
Romanian translation

### PR DESCRIPTION
**Pull request by Misaki290**
Romanian translation

### CODE REVIEW

**Comment by Misaki290** [comment]
Seems like I failed a check. 
Maybe I deleted something in the code.

**Comment by tripplehelix** [comment]
Great work, I'll look into the error later or tomorrow.

File: contribution/lang/ro.json [hunk 1]
```diff
@@ -0,0 +1,797 @@
...
+ 	 	 	},
+ 	 	 	"tut": {
+ 	 	 	 	"trans": "Dacă ți-a plăcut jocul, te rog să donezi:)",
+ 	 	 	 	"eng": "If you liked the game, please consider donation :)"
+ 	 	 	}
+ 	 	},
+ 	 	"printer": {
+ 	 	 	"name": {
+ 	 	 	 	"trans": "Imprimantă Moleculară 3D",
+ 	 	 	 	"eng": "Molecular 3d Printer
```
Comment (CynexXT) [inline contribution/lang/ro.json:48]: Here you forgot the closing ". Therefore the json becomes invalid.

Comment (Misaki290) [reply contribution/lang/ro.json:48]: Must have overlooked that. 😅

**Comment by DexterHuang** [comment]
the check is still failing XD

**Comment by Misaki290** [comment]
> the check is still failing XD

@tripplehelix  will look at the code tommorow.
Maybe I shouldn't translate the words that are in 
[ ] ? I really don't know what is the problem lol.

**Comment by CynexXT** [comment]
> the check is still failing XD

Well, the closing " is still missing in line 48...

**Comment by Misaki290** [comment]
> > the check is still failing XD
> 
> Well, the closing " is still missing in line 48...

I have corrected it but it's still failing. 😭

**Comment by tripplehelix** [comment]
OK I'll sort this now

**Comment by tripplehelix** [comment]
`"quest": {
 	 	"progression": {
 	 	 	"vars": [
 	 	 	 	"segmentDisplayName"
 	 	 	],
 	 	 	"trans": "Progresie-${progressionId}",
 	 	 	"eng": "Obține 3 Fragmente de Cheie diferite de la fiecare dungeon localizată în ${segmentDisplayName},\r\n                acestea sunt folosite pentru a crea cheia spre zona următoare.\r\n                Pentru a crea cheia, mergi la Terminal în [HUB]",
 	 	}
 	},`
 	
 	You've translated the wrong string, I'll reset it from the eng file.

**Comment by Misaki290** [comment]
> `"quest": { "progression": { "vars": [ "segmentDisplayName" ], "trans": "Progresie-${progressionId}", "eng": "Obține 3 Fragmente de Cheie diferite de la fiecare dungeon localizată în ${segmentDisplayName},\r\n acestea sunt folosite pentru a crea cheia spre zona următoare.\r\n Pentru a crea cheia, mergi la Terminal în [HUB]", } },`
> 
> ```
> You've translated the wrong string, I'll reset it from the eng file.
> ```

You're the best. ❤️

**Comment by tripplehelix** [comment]
> > `"quest": { "progression": { "vars": [ "segmentDisplayName" ], "trans": "Progresie-${progressionId}", "eng": "Obține 3 Fragmente de Cheie diferite de la fiecare dungeon localizată în ${segmentDisplayName},\r\n acestea sunt folosite pentru a crea cheia spre zona următoare.\r\n Pentru a crea cheia, mergi la Terminal în [HUB]", } },`
> > ```
> > You've translated the wrong string, I'll reset it from the eng file.
> > ```
> 
> You're the best. ❤️

I didn't reset anything. just added all the new words dex has been creating.

**Comment by Misaki290** [comment]
> > > `"quest": { "progression": { "vars": [ "segmentDisplayName" ], "trans": "Progresie-${progressionId}", "eng": "Obține 3 Fragmente de Cheie diferite de la fiecare dungeon localizată în ${segmentDisplayName},\r\n acestea sunt folosite pentru a crea cheia spre zona următoare.\r\n Pentru a crea cheia, mergi la Terminal în [HUB]", } },`
> > > ```
> > > You've translated the wrong string, I'll reset it from the eng file.
> > > ```
> > 
> > 
> > You're the best. ❤️
> 
> I didn't reset anything. just added all the new words dex has been creating.

I tried to look at what's wrong and I couldn't find the problem. 😅

**Comment by tripplehelix** [comment]
> > > > `"quest": { "progression": { "vars": [ "segmentDisplayName" ], "trans": "Progresie-${progressionId}", "eng": "Obține 3 Fragmente de Cheie diferite de la fiecare dungeon localizată în ${segmentDisplayName},\r\n acestea sunt folosite pentru a crea cheia spre zona următoare.\r\n Pentru a crea cheia, mergi la Terminal în [HUB]", } },`
> > > > ```
> > > > You've translated the wrong string, I'll reset it from the eng file.
> > > > ```
> > > 
> > > 
> > > You're the best. ❤️
> > 
> > 
> > I didn't reset anything. just added all the new words dex has been creating.
> 
> I tried to look at what's wrong and I couldn't find the problem. 😅

It was a couple of commas. enjoy translating the rest, if I was you I'd go through it all and make sure the translation is in the "trans": row and not the "eng": row

**Comment by tripplehelix** [comment]
eng is ignored and just used as a reference for translation.

**Pull request merged**